### DISCUSSION TITLE
error requires loader

### DISCUSSION THREAD

**Discussion by ValeryKuz** [gtm-support/vue-gtm#291] [General]
We are getting the following error, how this could be fixed?

` error  in ./node_modules/@gtm-support/vue2-gtm/dist/index.js

Module parse failed: Unexpected token (58:44)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file.`

**Comment by Shinigami92** [comment]
I assume you using webpack <= v4?
If yes, please use a modern version of webpack (>= v5) or e.g. try to switch to Vite
Otherwise just lower your version of `@gtm-support/vue2-gtm` to `1.3.0`

**Reply by ValeryKuz** [reply]
Tnx for your fast reply. Now we get

`TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator)) `

**Reply by Shinigami92** [reply]
That could mean everything, even not related to this plugin
Too less debug information to can help you further with that, I'm sorry :shrug:
